plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -2371,6 +2372,9 @@ public void dropView(ConnectorSession session, SchemaTableName viewName)
                 if (e.getErrorCode().equals(HIVE_VIEW_TRANSLATION_ERROR.toErrorCode())) {
                     // Ignore hive views for which translation fails
                 }
+                else if (e.getErrorCode().equals(HIVE_INVALID_VIEW_DATA.toErrorCode())) {
+                    // Ignore views that are not valid
```

Comment:
In this entire block we should be logging (ideally at WARN otherwise DEBUG).

Reply:
I'm not sure. This means every metadata call will log potentially many times, flooding the logs. You'll see the actual error if you try to query the view.